Sources/DatadogSDKTesting/Utils/PlatformUtils.swift

```diff
@@ -91,46 +91,22 @@ struct PlatformUtils {
 
     static func getAppearance() -> String {
         #if os(iOS) || os(tvOS) || os(watchOS)
-            let appearance: String
-            if #available(iOS 13.0, tvOS 13.0, *) {
-                switch UITraitCollection.current.userInterfaceStyle {
-                    case .unspecified:
-                        appearance = "unspecified"
-                    case .light:
-                        appearance = "light"
-                    case .dark:
-                        appearance = "dark"
-                    @unknown default:
-                        appearance = "unknown"
-                }
-            } else if #available(iOS 12.0, tvOS 12.0, *) {
-                switch UIScreen.main.traitCollection.userInterfaceStyle {
-                    case .dark:
-                        appearance = "dark"
-                    case .light:
-                        appearance = "light"
-                    case .unspecified:
-                        appearance = "unspecified"
-                    @unknown default:
-                        appearance = "unknown"
-                }
-            } else {
-                appearance = "light"
+            switch UITraitCollection.current.userInterfaceStyle {
+                case .unspecified: return "unspecified"
+                case .light: return "light"
+                case .dark: return "dark"
+                @unknown default: return "unknown"
```

Comment:
Question: are we removing support for old versions?

Reply:
Xcode 15+ dropped support for older targets. Or we drop old OS versions, or project for now should be developed in the Xcode 14. I don't use new features from Swift 5.9+, so can work with Xcode 14 if needed

Reply:
Make sense, thanks for the answer

Reply:
For sure on apple ecosystem is different but in .NET we are not allowed to deprecate versions almost never. So we end up with multiple sdks versions installed and building for key versions that are backward and forward compatible.

Reply:
Talked today with the Team - will try to support iOS 11+ for now, because it supported with Xcode 14.1 (latest Xcode supported by Apple Store). But we will have to figure out which version we will support in the future. Apple platforms doesn't allow you to setup a lot of SDK versions, they tied together to the Xcode version.